### PR TITLE
cmd/action: error message when using -w flag

### PR DESCRIPTION
**Pull request by yonidavidson**

### CODE REVIEW

File: cmd/action/apply.go [hunk 1]
```diff
@@ -47,6 +47,10 @@ func init() {
 
 // CmdApplyRun is the command used when running CLI.
 func CmdApplyRun(cmd *cobra.Command, args []string) {
+	if ApplyFlags.Web {
+		schemaCmd.PrintErrln("Open in UI server is not available in this release")
```
Comment (hilakashai) [inline cmd/action/apply.go:51]:
```suggestion
		schemaCmd.PrintErrln("Opening the UI server is not available in this release")
```

File: cmd/action/inspect.go [hunk 2]
```diff
@@ -40,6 +40,10 @@ func init() {
 
 // CmdInspectRun is the command used when running CLI.
 func CmdInspectRun(cmd *cobra.Command, args []string) {
+	if InspectFlags.Web {
+		schemaCmd.PrintErrln("Open in UI server is not available in this release")
```
Comment (hilakashai) [inline cmd/action/inspect.go:44]:
```suggestion
		schemaCmd.PrintErrln("Opening the UI server is not available in this release")
```

File: cmd/action/apply.go (hunk 1 above)
Comment (masseelch) [inline cmd/action/apply.go:51]:
```suggestion
		schemaCmd.PrintErrln("Opening the Graphical UI server is not available in this release")
```

Maybe also add instruction on how to get a release with the GUI.

File: cmd/action/inspect.go (hunk 2 above)
Comment (masseelch) [reply cmd/action/inspect.go:44]:
```suggestion
		schemaCmd.PrintErrln("Opening the Graphical UI server is not available in this release")
```

**Review by masseelch — APPROVED** [review]

**Review by rotemtam — APPROVED** [review]

**Pull request merged**